internal/envoy/auth/oauth2_filter.go

```diff
@@ -150,15 +153,15 @@ func NewFilterHTTPOAuth2(oauth2Options *options.OAuth2, args *parseAuthOptionsAr
 	authScopes := oauth2Options.AuthScopes
 	resources := oauth2Options.Resources
 
-	// Disable OAuth2 on the root path - "/" - see: <https://github.com/kubeshop/kusk-gateway/issues/680>.
-	passThroughMatcher := []*envoy_config_route_v3.HeaderMatcher{
-		{
-			Name: ":path",
-			HeaderMatchSpecifier: &envoy_config_route_v3.HeaderMatcher_ExactMatch{
-				ExactMatch: "/",
-			},
-		},
-	}
+	// // Disable OAuth2 on the root path - "/" - see: <https://github.com/kubeshop/kusk-gateway/issues/680>.
```

Comment:
make sure to remove this once you are done

Reply:
I've left it in as a comment. It's just useful to know why OAuth2 is being disabled on particular routes.

Hope that is OK?